### PR TITLE
Fixes #1728 | Add new member to credits

### LINKED ISSUE #1728: Credits: Add New ST member
Please add [my account](https://scratch.mit.edu/users/kittyloaf/) as a new ST member.

### PR DESCRIPTION
**Pull request by vutondesign**
Adds kchadha to the credits as requested in #1728.

### CODE REVIEW

File: src/views/credits/credits.jsx [hunk 1]
```diff
@@ -131,6 +131,11 @@ var Credits = React.createClass({
                         <img src="//cdn.scratch.mit.edu/get_image/user/2796185_170x170.png" alt="Julia Avatar" />
                         <span className="name">Julia Zimmerman</span>
                     </li>
+                    
+                     <li>
+                         <img src="//cdn2.scratch.mit.edu/get_image/user/27383273_60x60.png" alt="Karishma Avatar" />
+                         <span className="name">Karishma Chadhat</span>
```
Comment (thisandagain) [inline src/views/credits/credits.jsx]: This should be added in alphabetical order by last name. Also, it should be "Chadha" not "Chadhat". Thanks!

**Comment by vutondesign** [comment]
Oops, that should be it now.

File: src/views/credits/credits.jsx [hunk 2]
```diff
@@ -26,6 +26,11 @@ var Credits = React.createClass({
                         <img src="//cdn.scratch.mit.edu/get_image/user/3581881_170x170.png" alt="Carl Avatar" />
                         <span className="name">Carl Bowman</span>
                     </li>
+                                        
+                     <li>
```
Comment (thisandagain) [inline src/views/credits/credits.jsx]: Looks great! One last issue: It looks like your indentation is off by one space.

Comment (vutondesign) [reply src/views/credits/credits.jsx]: Fixes 😄

File: src/views/credits/credits.jsx [hunk 3]
```diff
@@ -26,6 +26,11 @@ var Credits = React.createClass({
                         <img src="//cdn.scratch.mit.edu/get_image/user/3581881_170x170.png" alt="Carl Avatar" />
                         <span className="name">Carl Bowman</span>
                     </li>
+                                        
+                    <li>
+                         <img src="//cdn2.scratch.mit.edu/get_image/user/27383273_60x60.png" alt="Karishma Avatar" />
```
Comment (thisandagain) [inline src/views/credits/credits.jsx]: These two inner elements are still off by one space.

Comment (vutondesign) [reply src/views/credits/credits.jsx]: Okay, sorry I didn't catch these.

**Comment by vutondesign** [comment]
No Problem 😄

**Pull request merged**